src/pybind/mgr/nfs/tests/test_nfs.py

```diff
@@ -896,17 +941,50 @@ def _do_test_update_export_with_list(self):
         assert export.clients[0].access_type is None
         assert export.cluster_id == self.cluster_id
 
+    def _do_test_update_export_cephfs(self):
```

Comment:
this isn't called anywhere, seems you missed adding a test case calling `self._do_mock_test()`?

Reply:
Yes, thanks for noticing, fixed that now!